packages/forma-36-website/src/pages/guidelines/copy/grammar/punctuation/index.mdx

```diff
@@ -0,0 +1,40 @@
+# Punctuation
+
+## Use periods consistently
+
+Keep UX copy scannable by eliminating unnecessary periods.
+Do not use periods on:
+
+- buttons
+- labels
+- tooltips
+- bulleted lists
+
+Use periods on:
+
+- multiple sentences
+- any sentence followed by a link
+- modal body text
+
+![Use periods consistently](/use-periods-consistently-1.jpg)
+
+![Use periods consistently](/use-periods-consistently-2.jpg)
+
+## Avoid exclamation marks
+
+Exclamation marks may come across as shouting and do not match the brand tone established for the web app. There are some exceptions when it is appropriate to use exclamation marks, such as a greeting or congratulatory message.
```

Comment:
Suggest rewording to not explicitly mention the web app but instead "Contentful products"